### PR TITLE
phony OCI image bundle

### PR DESCRIPTION
**Pull request by ryantm**
Why
===
* We'd like to be able to quickly build containers that depend on nixmodules.

What changed
===
* Add new bundle-phony-ocis package output that makes a phony OCI image for each nixmodule. The OCI images contain a layer that is the /nix/store paths of the module, and a ZTOC blob for that layer. These modules cna be uploaded to a Registry.

Test plan
===
* built `nix build .#bundle-phony-ocis` output looks reasonable
* can push output to Registry

Rollout
=======

_Describe any procedures or requirements needed to roll this out safely (or check the box below)_

- [x] This is fully backward and forward compatible

### CODE REVIEW

File: pkgs/mk-phony-oci/builder.sh [hunk 1]
```diff
@@ -0,0 +1,81 @@
+set -eou pipefail
+. .attrs.sh
```
Comment (cdmistman) [inline pkgs/mk-phony-oci/builder.sh:2]: where's this file?

Comment (cdmistman) [reply pkgs/mk-phony-oci/builder.sh:2]: oh nevermind, it's a stdenv file

Comment (ryantm) [reply pkgs/mk-phony-oci/builder.sh:2]: It gets created via `__structuredAttrs = true;`

**Review by cdmistman — APPROVED** [review]
no way to test as i deleted my devvm but LGTM

**Pull request merged**